lib/wor/paginate/formatter.rb

```diff
@@ -46,6 +51,22 @@ def serialized_content
       def serializer
         options[:each_serializer]
       end
+
+      def next_page_url
+        return nil unless next_page
+
+        replace_query_params(current_url, page: next_page)
+      end
+
+      def previous_page_url
```

Comment:
This logic is the same to the previuos... What if you move to another method and group the logic?
```
      def next_page_url
        replace_query_url(next_page)
      end

      def previous_page_url
        replace_query_url(previous_page)
      end

      private

      def replace_query_url(page)
        return nil unless page

        replace_query_params(current_url, page: page)
      end
```